format/puffin-spec.md

```diff
@@ -123,6 +123,44 @@ The blob metadata for this blob may include following properties:
 
 - `ndv`: estimate of number of distinct values, derived from the sketch.
 
+#### `delete-vector-v1` blob type
+
+A serialized delete vector that represents the positions of rows in a file that
+are deleted.  A set bit at position P indicates that the row at position P is
+deleted.
+
+The bitmap supports positive 64-bit positions, but is optimized for cases where
```

Comment:
what does "positive 64-bit positions" exactly mean?
does it mean
positions in range 0..(2^64-1) (unsigned 64-bit), or 
positions in range 0..(2^63-1) (positive part of 64-bit integers)

(also, zero is not a positive number, but i assume zero is addressable)

Reply:
Does it matter? These positions are row positions. There is never going to be a file with that many rows. Many, many other things would fail first.

Reply:
I've clarified that the MSB must always be 0.

Reply:
> I've clarified that the MSB must always be 0.

thanks